packages/ui/src/components/icons/IconOutlineX.tsx

```diff
@@ -1,22 +1,27 @@
 import React from 'react'
 import { IconProps } from './types'
 
-export const IconOutlineX = ({ size = '1em', ...props }: IconProps) => {
+export const IconOutlineX = ({
+	size = '1em',
+	strokeWidth = '4',
```

Comment:
Are these changes still needed? I think ideally we would revert these changes so that this icon doesn't perform differently than the other icons.

Reply:
Per the design IconOutlineX shall have strokeWidth of 8. How else can I do the same?

Reply:
Good question! I think it should already be matching the styles of the other modal? I don't believe we are using `IconOutlineX`, but sorry if I am missing something